tests/Oryx.Integration.Tests/Fixtures/MySqlDbContainerFixture.cs

```diff
@@ -56,7 +56,7 @@ protected override void InsertSampleData()
         {
             const string sqlFile = "/tmp/mysql_setup.sql";
             var sqlQuery = GetSampleDataInsertionSql();
-            var dbSetupScript = new ShellScriptBuilder()
+            var dbSetupScript = new ShellScriptBuilder(null, false)
```

Comment:
nit: why does this constructor need the additional arguments now?

Reply:
+1

Reply:
@waliMSFT do you mind pointing to this method that uses the staging account as default? Just my two cents that it's more intuitive to make this change in this method - `AddDefaultTestEnvironmentVariables` than introduce new arguments to this constructor - `ShellScriptBuilder` to determine if we want to use the staging account or not.